Wording changes to docs

maikhoepfel opened the pull request:
I eagerly read through the (excellent, thanks!) documentation for channels, and had to re-read one or two sentences. So being a good citizen, I'm suggesting a few fixes here and there.

andrewgodwin commented on docs/concepts.rst:
```diff
@@ -5,13 +5,13 @@ Django's traditional view of the world revolves around requests and responses;
 And, beyond that, there are plenty of non-critical tasks that applications
 could easily offload until after a response as been sent - like saving things
-into a cache, or thumbnailing newly-uploaded images.
+into a cache, sending emails or thumbnailing newly-uploaded images.
```
I explicitly left off "sending emails" as people usually route this through a reliable system, and channels deliberately isn't. Perhaps modify to add `(with a retry mechanism)`?

maikhoepfel replied:
That makes sense. I'll remove it to avoid scope creep for the PR. I do think it would be helpful to have a list of common task, and mentioning how well they could be achieved with channels.

andrewgodwin commented on docs/concepts.rst:
```diff
@@ -120,22 +117,20 @@ In fact, this is how Channels works. The interface servers transform connections
 on model saves, on other incoming messages, or from code paths inside views
-and forms.
+and forms. That approach comes in handy for push-style
+code - where you use HTTP2's server-sent events or a WebSocket to notify
```
HTTP2 server-side events do not actually allow arbitary notification of the client; they're more a way to push resources (images, css) to the browser before it requests them.

maikhoepfel replied:
Ah of course, you're right. But I just moved the sentence around, because I found it easier to understand that way. Did both of us mean [HTML5 server-side events](http://www.w3schools.com/html/html5_serversentevents.asp) instead of HTTP2?

andrewgodwin commented on docs/concepts.rst:
```diff
@@ -240,15 +235,15 @@ Next Steps
-persistence like Celery, or you'll need to make a management command that
-checks for completion and re-submits a message to the channel if nothing
-is completed (rolling your own retry logic, essentially).
+One thing channels do not, however, is guaranteeing delivery. If you need
```
Perhaps just "guarantee" here?

maikhoepfel replied:
Whoops, I meant to write that. Thanks.

andrewgodwin commented on docs/getting-started.rst:
```diff
@@ -233,7 +240,7 @@ Persisting Data
 skirting around the real design pattern - persistent state for connections.
 Let's consider a basic chat site where a user requests a chat room upon initial
-connection, as part of the query string (e.g. ``http://host/websocket?room=abc``).
+connection, as part of the query string (e.g. ``https://host/websocket?room=abc``).
```
I was wrong originally - the protocol here should be `wss`

maikhoepfel replied:
Fixed.

andrewgodwin commented on docs/getting-started.rst:
```diff
@@ -451,7 +462,7 @@ command run via ``cron``. If we wanted to write a bot, too, we could put its
 pass through it.
 
-Linearization
+Linearisation
```
I've changed my mind on UK vs US English - if this is going to be merged into Django, it needs to use US English, so probably need to keep the z. Sorry!

maikhoepfel replied:
No worries.

maikhoepfel commented:
Thanks for the quick review, and sorry for the late response. I pushed a new commit, hopefully addressing your comments. Could you have another look?

andrewgodwin commented:
Perfect, thanks!
